### ISSUE TITLE
Members profile, in their photo section not able to display more after the first images are loaded

### ISSUE THREAD

**Issue by d1a8lo24** [moxi9/phpfox#413]
When you browse to a member's profile, and see their section for photos, it will load the albums and the fist set of images. once you start scrolling down and the load the next set of pictures is activated, the images get loaded but they are not displayed the page does grow as more pictures are being loaded but the images are still hidden.

Also the loading image stays there.

**Comment by Data66PF** [comment]
Are you seeing this issue at the live demo, http://community.phpfox.com ? How many photos does the user have? I'm testing at the demo but maybe I don't have enough photos loaded.

**Comment by d1a8lo24** [comment]
If a member has more than 20 images, so the first 20 will show without a problem once you start scrolling down and you trigger the function to load more images. Now the images load but they don't show they are still hidden, and you know they are loading because the page grows. also the little image loader that has a blue highlight that is telling you that something is loading will stay there.

Tested it on FF, IE, Chrome, and safari.

This was tested on the latest developer's version v407 on a local host.

And I am pretty sure is on the live demo as I found many small problems that I found on the local installation.
